source/pages/Settings/ConnectHardwareWallet.tsx

```diff
@@ -11,20 +11,38 @@ import { getController } from 'utils/browser';
 const ConnectHardwareWalletView: FC = () => {
   const [selected, setSelected] = useState<boolean>(false);
   const [isTestnet, setIsTestnet] = useState<boolean>(false);
+  const { activeNetwork, isBitcoinBased, accounts } = useSelector(
+    (state: RootState) => state.vault
+  );
 
-  const controller = getController();
+  const trezorAccounts = Object.values(accounts.Trezor);
 
-  const handleCreateHardwareWallet = async () => {
-    await controller.wallet.account.sys.trezor.createAccount();
+  const getSlip44 = () => {
+    let slip44 = '';
+
+    switch (isBitcoinBased) {
+      case true:
+        slip44 = '57';
```

Comment:
This is only for syscoin we probably should add the blockchain id by the connected chain ID and exclude utxo testnets in the isBitcoinBased case